data/common.yaml

```diff
@@ -1 +1,16 @@
---- {}
+---
+
+profile_gpu::dcgm::install::bind_mount_install:        # See reference.md for when this is needed, set to undef on purpose
```

Comment:
I think this rebase is not correct. You mentioned that you switched this to be `null`, but it is still showing like it was originally.

Reply:
Weird, if I browse to the file via the web manually it does have the null. Also looks ok when I pull down the repo fresh in a new location on my local machine. Does it look ok if you do either of those?

I guess since the contents of the commits seem to differ instead of just # of commits maybe creating a new merge request is safer. Tried some googling but didn't see anyone with this behavior, doesn't help that the keywords floods the results with other more basic questions/problems.